content/topics/intro-to-relational-databases/_index.md

```diff
@@ -1,15 +1,69 @@
 ---
-title: SQL
-ready: True
+title: Introduction to relational database
+ready: true
 ---
 
-## Tutorial
+# INTRODUCTION TO RELATIONAL DATABASE 
 
-Complete the DataCamp Intro and Intermediate SQL courses:
+**LESSONS**
 
+- Lesson 1: 
+  > Data and Tables:
+    * learn the principles behind relational data organization:tables,queries,aggregations,
+    keys and joins.
+- Lesson2:
+    > Elements of **SQL**
+    * start learning  by using **select** & **insert** statements/commands to read and write
+     data to database tables
+- Lesson3: 
+  > Nodejs SQL-API: 
+    * learn SQL-APIs and  apply your knowledge to fix common bugs that arise from database-backend-web service
+  > Python SQL-API: 
+    * learn SQL-APIs and  apply your knowledge to fix common bugs that arise from database-backend-web service
+
+- Lesson4:
+  > Deeper Into SQL 
+## What a Relational Database Is:
+A relational database is a type of database that stores and provides access to data points that
+are related to one another. Relational databases are based on the relational model, an intuitive,
+straightforward way of representing data in tables. In a relational database, each row in the table
+is a record with a unique ID called the key. The columns of the table hold attributes of the data,
+and each record usually has a value for each attribute, making it easy to establish the relationships
+among data points.
+
+## Understanding difference between Realtion & Non relational database:
+**Structure vs Non structured query language**: 
+>Read more about SQL & NoSql
+* [click here --> SQL & NoSql](https://www.youtube.com/watch?v=ZS_kXvOeQ5Y)
+
+
+>e.g. 
+>![alt text](https://miro.medium.com/max/1512/1*LEksJP5OtS8GEBdd2Jy4WQ.png)
+>Also See: [What is RDBMS](https://www.youtube.com/watch?v=h8IWmmxIyS0)
+
+
+## Best practices  
+  *(N.B)* Always do not forget to include your .gitignore file,which will have contain you files want to exclude git pushing into remote repository and commit that change before you add your Otherwise, you run the risk of committing an early version of your .env to source control.
+ ```
+//.gitignore examples
+FOR PYTHON
+    &
+FOR Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+FOR VIM
+*.swp
+*.swo
+ ````
+
+**Resouces**
+- [Intro to RDBMS in Javascript](https://www.youtube.com/watch?v=QExydkuuE30)
 - [Intro to SQL for data science](https://www.datacamp.com/courses/intro-to-sql-for-data-science)
 - [Intermediate SQL](https://www.datacamp.com/courses/intermediate-sql)
+- [Java / SQL ](https://www.youtube.com/watch?v=hXxzn27D9s4)
 
-## Assignment
+**Assignment**
 
-See the SQL assignment in Projects: {{% contentlink "projects/SQL" %}}
+See Intro-to-relational-database **Assignment**  in Projects: 
```

Comment:
This is a topic, not an assignment. The project should link here, not the other way around. 

Also you should learn how the contentlink shortcode works. It is clear from this that you are guessing